### DISCUSSION TITLE
Welcome to Nord Discussions!

### DISCUSSION THREAD

**Discussion by Tetrax-10** [Tetrax-10/Nord-Spotify#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RetardedNConfused** [comment]
is that the washing machine girl from K/DA?

**Reply by Tetrax-10** [reply]
Lol, washing machine girl? Idk, just found that pic on the internet.
